feat: redo landing page

cabreraalex opened the pull request:
Adds a new video, testimonials, and updates structure slightly.

fix ZEN-224

cabreraalex committed:
feat: redo landing page

Sparkier reviewed (approved):
Good to merge from my side since it's already much better compared to before. I have some additional suggestions, though:

- Video: Really like it overall, I think it's a huge improvement.
  - Can we start with a Zeno screen? This would also act as a separator between loops. Something like this: https://github.com/zeno-ml/zenoml.com/assets/5690524/1dac49d7-6cb8-453b-b068-963d9ef1593c
  - I kinda dislike the purple gradient transition slides. Would it look better if we made them white with purple text or completely purple?
  - The video has a border top and bottom but is cut off left and right. In Screen Studio you can set a resolution for the browser window when recording. Maybe set that to something like 1920 or better 3840 before recording to make sure the aspect is right?
  - The last transition slide is shown too short for the amount of text. I feel like this is the most important message we could show it a bit longer.
  - Can we put the video in a div with rounded corners? This is the only container that does not have them.
- Not sure about the gradient of the top element following the cursor especially since it jumps around once you hover the video or one of the buttons. Generally feel like either no change or an automatic one over time.
- Can we add company logos to the endorsements?
- Should we move sponsors to the footer?
- Can we swap docs and about in the footer as well?

cabreraalex committed:
updates

cabreraalex committed:
updates

cabreraalex commented:
Updated the video slides (didn't update the actual interaction videos), changed the top gradient to be a slow shift, added logos, swapped footer.

I think sponsors okay where it is for now?

Sparkier commented:
two more nits about the video:
1. it seems like it does not auto-play on chrome
2. is there a way to show the video controls only on hover? They show for me when the video starts playing by default which is somewhat ugly.

Also do we want to rename "Testimonials"? Maybe "User Feedback" or something like that? Might be just me but testimonials to me sounds more like we asked people to say nice things.

cabreraalex committed:
hide use cases
